### ISSUE TITLE
Create Tools to Generate Marketing Campaigns URL to Share 

### ISSUE THREAD

**Issue by dirtyhenry** [Bootstragram/buckygem#4]
As part of the marketing stuff that everyone needs in its toolbox, we need to provide a convenient way to make sharing URL easily while being able to track the campaign for both Google Analytics & iTunes Connect.

Sources:
- [Google Analytics URL Builder](https://support.google.com/analytics/answer/1033867?hl=en)
- [iTunes Connect Campaign Link Generator](https://analytics.itunes.apple.com/#/campaigngenerator?app=1021607999) (for Statium iOS)

**Comment by dirtyhenry** [comment]
In progress but I'm having a hard time deciding if we should encode spaces characters into '%20' or '+'. The debate is not clear on internet: SO people seem to say '+' is better for the query part of an URL whereas Apple and Google escapes them as %20 in their tools.

**Comment by zertyuio** [comment]
Do you know ?
https://support.google.com/analytics/answer/1033867

google web tool to generate campaign url

**Comment by dirtyhenry** [comment]
Yes, it's one of the links in the issue description. But I want a tool to build the URL in the command line. 
I don't think Google's can do that.
